sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py

```diff
@@ -1130,40 +484,6 @@ def deserialization_callback(raw_response, _, headers):  # pylint: disable=unuse
             ),
         )
 
-    @distributed_trace
-    def get_translation_status(self, translation_id: str, **kwargs: Any) -> TranslationStatus:
-        """Gets the status of the translation operation.
-
-        Includes the overall status, as well as a summary of
-        the documents that are being translated as part of that translation operation.
-
-        :param str translation_id: The translation operation ID.
-        :return: A TranslationStatus with information on the status of the translation operation.
-        :rtype: ~azure.ai.translation.document.TranslationStatus
-        :raises ~azure.core.exceptions.HttpResponseError or ~azure.core.exceptions.ResourceNotFoundError:
-        """
-
-        translation_status = self._client.get_translation_status(translation_id, **kwargs)
-        return TranslationStatus._from_generated(  # pylint: disable=protected-access
-            _TranslationStatus(translation_status)  # type: ignore[call-overload]
-        )
-
-    @distributed_trace
-    def cancel_translation(self, translation_id: str, **kwargs: Any) -> None:
-        """Cancel a currently processing or queued translation operation.
-
-        A translation will not be canceled if it is already completed, failed, or canceling.
-        All documents that have completed translation will not be canceled and will be charged.
-        If possible, all pending documents will be canceled.
-
-        :param str translation_id: The translation operation ID.
-        :return: None
-        :rtype: None
-        :raises ~azure.core.exceptions.HttpResponseError or ~azure.core.exceptions.ResourceNotFoundError:
-        """
-
-        self._client.cancel_translation(translation_id, **kwargs)
-
```

Comment:
I think we need to keep cancel_translation in the _patch. The return type has changed when the generated code is used. 

GA --> returns None
this version --> returns TranslationStatus

Reply:
Added back